packages/worker/src/interceptors.ts

```diff
@@ -85,6 +86,68 @@ export interface ActivityInboundCallsInterceptorFactory {
   (ctx: ActivityContext): ActivityInboundCallsInterceptor;
 }
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Nexus Interceptors
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export type NexusInterceptorsFactory = (ctx: nexus.OperationContext) => NexusInterceptors;
+
+/**
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export type NexusInterceptors = {
+  inbound?: NexusInboundCallsInterceptor;
+  outbound?: NexusOutboundCallsInterceptor;
+};
+
+/**
+ * A function that takes a Nexus Context and returns an interceptor
+ *
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export type NexusInboundCallsInterceptor = {
+  execute?: (
+    input: NexusExecuteInput,
+    next: Next<NexusInboundCallsInterceptor, 'execute'>
+  ) => Promise<NexusExecuteOutput>;
+};
+
+/**
+ * Input for NexusInboundCallsInterceptor.execute
+ *
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export interface NexusExecuteInput {
+  readonly args: unknown[];
+  readonly headers: Headers;
+}
+
+/**
+ * Output for NexusInboundCallsInterceptor.execute
+ *
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export interface NexusExecuteOutput {
+  readonly result: unknown;
+}
+
+/**
+ * A function that takes a Nexus Context and returns an interceptor
```

Comment:
As above: I don't understand this comment

Reply:
Removed description on that specific typedoc. It should have one, but so should all other interceptor-related types, but most don't. I'll revisit in a distinct PR.